### DISCUSSION TITLE
What is used of ... (3 dots)in dart

### DISCUSSION THREAD

**Discussion by gajkc** [community/community#28315] [Mobile] [answered]
What is used of ... (3 dots)in dart

**Comment by sthakrishnakumar** [comment, marked as answer]
Hello @gajkc ,

In dart, triple dot you are mentioning is called Spread Operator.
Spread operator are used in :
-list
-sets
-map

Here, i will provide some reference to use Spread Operator in list.
For example,

     List<int> list1 = [ 0,1,2 ];
     List<int> list2 = [ 5,6,7 ];
    
     List<int> combine = [ ...list1, ...list2, 3,3 ];
     
    print(combine);

    output will be [ 0,1,2,5,6,7,3,3 ]

   For more reference, you can visit this url : https://www.woolha.com/tutorials/dart-using-triple-dot-spread-operator-examples

  Hope this answer helped you. Thank You!

**Reply by gajkc** [reply]
Thank you
